src/webgl/p5.RendererGL.js

```diff
@@ -1507,6 +1507,15 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     this.GL.clear(this.GL.COLOR_BUFFER_BIT | this.GL.DEPTH_BUFFER_BIT);
   }
 
+  /**
+   * Resets all depth information so that nothing previously drawn will
+   * occlude anything subsequently drawn.
+   */
+  clearDepth() {
+    this.GL.clearDepth(1);
```

Comment:
Is there ever a scenario where someone would want to use a value other than 1? If so we could provide 1 as a default and optionally let the user override with their own.

Reply:
Makes sense! we can give it an optional parameter that gets defaulted to 1.